geode-core/src/integrationTest/java/org/apache/geode/cache/RegionExpirationIntegrationTest.java

```diff
@@ -84,15 +85,16 @@ public void increaseRegionTtl() throws Exception {
         .setRegionTimeToLive(new ExpirationAttributes(secondTtlSeconds, DESTROY));
 
     await().until(region::isDestroyed);
-    assertThat(NANOSECONDS.toSeconds(nanoTime() - startNanos))
-        .isGreaterThanOrEqualTo(secondTtlSeconds);
+    Instant endInstant = Instant.now();
+    assertThat(Duration.between(startInstant, endInstant))
+        .isGreaterThanOrEqualTo(Duration.ofSeconds(secondTtlSeconds));
   }
 
   @Test
-  public void decreaseRegionTtl() throws Exception {
+  public void decreaseRegionTtl() {
     int firstTtlSeconds = 5;
```

Comment:
For readability I wonder if this is clearer now that we have introduced these new time classes:
```java
final Duration firstTtl = Duration.ofSeconds(5);
final Duration secondTtl = Duration.ofSeconds(1);
...
regionFactory.setRegionTimeToLive(new ExpirationAttributes(firstTtl.getSeconds(), DESTROY));
...
assertThat(Duration.between(startInstant, endInstant))
         .isLessThan(firstTtlSeconds);
```